generate/generate_check_test.go

```diff
@@ -28,14 +29,16 @@ func TestCheckReturnsOutdatedStackFilenames(t *testing.T) {
 	stack1 := s.CreateStack("stacks/stack-1")
 	stack2 := s.CreateStack("stacks/stack-2")
 
-	stack1Dir := stack1.RelPath()
-	stack2Dir := stack2.RelPath()
+	stack1val, err := tmstack.Load(s.RootDir(), stack1.Path())
+	assert.NoError(t, err)
+	stack2val, err := tmstack.Load(s.RootDir(), stack2.Path())
+	assert.NoError(t, err)
```

Comment:
Maybe something like this, leveraging the sandbox:

```go
	stack1 := s.LoadStack(stackEntry1.Path())
	stack2 := s.LoadStack(stackEntry2.Path())
```

Now that the test also has stack instances we could call the sandbox stack an "stackEntry", or else it is a little confusing.